src/components/ux-dropdowns/ux-dropdowns.feature

```diff
@@ -0,0 +1,6 @@
+Feature: ux-dropdowns test suite
+
+	Scenario: Loading ux-dropdowns
+		Given I have loaded component "ux-dropdowns" with use case "linkDropdown"
+		Then the element "button" should be displayed
```

Comment:
These 2 test statements can be rewritten like so:

		Then there will be an element for "button"
		And the element "button" will have the class "button"

This will reuse the test cases listed in /src/support/steps.js, so that you don't need to rewrite the test case inside ux-dropdowns.steps.js